python/pyproject.toml

```diff
@@ -22,22 +22,26 @@ classifiers = [
 ]
 requires-python = ">=3.10"
 dependencies = [
-  # pathfinder
+  # pathfinder for finding CUDA libraries
   "cuda-pathfinder",
-
-  # Library expects to find shared libraries
-  # libcupti, libnvperf_target, libnvperf_host
-  # pathfinder is used to find it in the Python layout
-  "nvidia-cuda-cupti-cu12",
-
-  # The shared library
-  # libnvidia-ml must be installed system-wide
-  # (Debian package provider: libnvidia-compute)
 ]
 dynamic = ["version"]
 readme = { file = "README.md", content-type = "text/markdown" }
 
 [project.optional-dependencies]
+# CUDA 12.x dependencies
+cu12 = ["cuda-bindings>=12.0.0,<13.0.0", "nvidia-cuda-cupti-cu12"]
```

Comment:
Remove the `cuda-bindings` dependency as it's just used as a proxy for the CUDA major version right now.